Adding Initial Commint for Shipa

ravilach opened the pull request:
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [y ] Have you followed the guidelines in our Contributing document in setting up the application?
* [y ] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [y ] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [y ] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
![image](https://user-images.githubusercontent.com/42775804/150397513-910acf06-3dd8-45bf-94ae-a495903bea97.png)

saiyam1814 commented on shipa/post_install.md:
```diff
@@ -0,0 +1,54 @@
+
+### Wiring
+Shipa for itself installs Nginx as a default ingress controller for the control plane. If
+leveraging Civo, make sure as of Shipa 1.5.2, to unselect Nginx from the ingrress controllers
```
ingress spellings, also civo kubernetes comes with Traefik and not nginx by default so which one a user needs ? the default traefik one would work or they need to deselect that as well ?

ravilach replied:
It's just a heads up, the Shipa Control Plane can't be installed as of yet with an existing Nginx installation as Nginx is the ingress controller between the Control Plane and Shipa Nodes. Having Traefik should be ok I would need to test it. Also what's misspelled (spelling and I are not friends ha)

ravilach replied:
Oh I see it

ravilach replied:
So with testing, doesn't look like Civo can support two external IP addresses, or at least I couldn't figure out how to have more than one external IP with spinning up just a Civo Cluster. WIll advise unselecting Traefik for now. 
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/42775804/151385518-6482e85a-f103-4c1c-b42c-a5cc95b09372.png">

saiyam1814 commented on shipa/install.sh:
```diff
@@ -0,0 +1,17 @@
+kubectl wait --for=condition=available --timeout=300s deployment/cert-manager-webhook  -n cert-manager
+
+#Install Shipa
+helm upgrade --install shipa shipa-charts/shipa \
```
So this installs shipa agent or something ? because Shipa as such is not open source right ?

ravilach replied:
This is the Shipa Control Plane. It installs the free version by default or a user can supply a license.

ravilach commented:
@saiyam1814 thanks for the review. Corrected the spelling and changed the verbiage to remove all default ingress controllers. Was able to deploy. 
<img width="816" alt="image" src="https://user-images.githubusercontent.com/42775804/151388165-626eb548-6e1a-4497-b8b6-a3c462ef83d8.png">

saiyam1814 reviewed (approved):
LGTM
